integrated_channels/xapi/migrations/0011_alter_xapilearnerdatatransmissionaudit_index_together.py

```diff
@@ -27,11 +40,7 @@ class Migration(migrations.Migration):
                     ),
                 ],
                 database_operations=[
-                    migrations.RunSQL(sql="""
-                        CREATE INDEX xapi_xldta_85936b55_idx
-                        ON xapi_xapilearnerdatatransmissionaudit (enterprise_customer_uuid, plugin_configuration_id)
-                        ALGORITHM=INPLACE LOCK=NONE
-                    """, reverse_sql="""
+                    migrations.RunSQL(sql=generate_index_sql(db_engine), reverse_sql="""
```

Comment:
Nit: I think this is okay either way, but if it's only being used once, I think you might as well inline it like you did elsewhere in this PR.

Reply:
noted